lib/Twinkly.js

```diff
@@ -36,7 +36,11 @@ class Twinkly {
         return this.requestService.get("gestalt", false)
             .then(json => {
                 this.name = json.device_name;
-                this.uuid = json.uuid;
+                if (json.uuid == "00000000-0000-0000-0000-000000000000") {
+                    this.uuid = `00000000-0000-1000-8000-${json.mac.replaceAll(":", "")}`;
```

Comment:
I would suggest to just use zeroes instead of the '1000' and the '8000' sequence in the time-stamp part, and ignore 'versions' etc.